data/sql_setup/prepare_schedule_a.sql

```diff
@@ -1,12 +1,7 @@
--- Create table to hold sub_ids of records that fail during the nightly
--- processing so that they can be tried at a later time.
--- The "action" column denotes what should happen with the record:
---    insert, update, or delete
+-- Drop the old nightly refresh retry table and function if they still exist.
 drop table if exists ofec_sched_a_nightly_retries;
```

Comment:
Yes, I can file one after the next release goes out to make sure this is propagated through all environments.